provider/ansible.py

```diff
@@ -132,6 +133,12 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # On s390x and ppc64le, the ansible dependency cryptography needs
```

Comment:
Package installation should not be coupled to the architectures, If we have other packages that want to install, it makes sense to make it configurable. For instance, some pkgs we want to prepare and avoid to be installed via pip.